src/components/rb_search_widget.module.scss

```diff
@@ -8,6 +8,9 @@ fieldset {
   border: 1px solid rgba(193, 193, 193, 0.95);
   border-radius: 5px;
   background-color: #F6F8FA;
+  width: max-content;
```

Comment:
Please take note and ask Harald if it's ok to have a dynamic width of the widget, because it will change depending on the content. Not sure if that's what we want.